### PR TITLE
Update import_test.go (handle error properly)

### PR DESCRIPTION
**Pull request by TheInvincibleRalph**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/28393

```release-note
NONE
```

### CODE REVIEW

File: server/channels/api4/import_test.go [hunk 1]
```diff
@@ -144,6 +149,8 @@ func TestImportInLocalMode(t *testing.T) {
 
 	cnt2, err := th.App.Srv().Store().Post().AnalyticsPostCount(&model.PostCountOptions{UsersPostsOnly: true})
 	require.NoError(t, err)
-	// Just a sanity check to ensure new posts are actually added in the system.
+
+	// Sanity check to ensure new posts are actually added in the system.
 	require.Greater(t, cnt2, cnt1)
 }
+
```
Comment (hanzei) [inline server/channels/api4/import_test.go:156]: Unrelated change?

File: server/channels/api4/import_test.go [hunk 2]
```diff
@@ -144,6 +149,8 @@ func TestImportInLocalMode(t *testing.T) {
 
 	cnt2, err := th.App.Srv().Store().Post().AnalyticsPostCount(&model.PostCountOptions{UsersPostsOnly: true})
 	require.NoError(t, err)
-	// Just a sanity check to ensure new posts are actually added in the system.
+
+	// Sanity check to ensure new posts are actually added in the system.
```
Comment (hanzei) [inline server/channels/api4/import_test.go:153]: Unrelated change?

**Review by hanzei — COMMENTED** [review]
Let's see what CI reports

File: server/channels/api4/import_test.go (hunk 1 above)
Comment (TheInvincibleRalph) [reply server/channels/api4/import_test.go:156]: Fixed.

File: server/channels/api4/import_test.go (hunk 2 above)
Comment (TheInvincibleRalph) [reply server/channels/api4/import_test.go:153]: Fixed

File: server/channels/api4/import_test.go [hunk 3]
```diff
@@ -144,6 +149,7 @@ func TestImportInLocalMode(t *testing.T) {
 
 	cnt2, err := th.App.Srv().Store().Post().AnalyticsPostCount(&model.PostCountOptions{UsersPostsOnly: true})
 	require.NoError(t, err)
+
```
Comment (hanzei) [inline server/channels/api4/import_test.go:152]: Unrelated change

File: server/channels/api4/import_test.go
Comment (hanzei) [inline server/channels/api4/import_test.go:1]: Can you please remove `channels/api4/import_test.go` from the ignore list of golangci-lint?

Comment (TheInvincibleRalph) [reply server/channels/api4/import_test.go:1]: Removed already. [Here](https://github.com/mattermost/mattermost/pull/28428/commits/9e6cee1c7b80a3d38d80b96caf90b80e52846f95)

Comment (hanzei) [reply server/channels/api4/import_test.go:1]: Can you please do that as part of this PR? Let's keep every PR self-contained.

**Review by hanzei — CHANGES_REQUESTED** [review]

**Pull request closed without merging**